scripts/feature_tests/upgrade/Makefile

```diff
@@ -1,11 +1,19 @@
-M3PATH := "$(dirname "$(readlink -f "${0}")")/../../"
-# This needs to be moved to upgrade.sh and different value should be set before
-# each use case.
-# export NUM_OF_MASTER_REPLICAS := 1
-# export NUM_OF_WORKER_REPLICAS := 1
+M3PATH := "$(dirname "$(readlink -f "${0}")")../../../"
+export NUM_OF_MASTER_REPLICAS := 3
+export NUM_OF_WORKER_REPLICAS := 1
 
-all: upgrade
-
-.PHONY: upgrade
+all: fetch_manifests upgrade
+#all: provision fetch_manifests pivoting upgrade repivoting deprovision
+provision:
+	./../feature_test_provisioning.sh
+fetch_manifests:
+	./../../fetch_manifests.sh
+pivoting:
+	./../pivoting/pivot.sh
```

Comment:
Do you need these `pivoting` and `repivoting` make target here or is it enough to call them from the `Makefile` of pivoting when needed ?

Reply:
removed.